adapter/client/callback_test.go

```diff
@@ -0,0 +1,73 @@
+package client
+
+import (
+	"net/url"
+	"testing"
+
+	"github.com/ibm-cloud-security/app-identity-and-access-adapter/tests/fake"
+)
+
+func TestCallbackURLForTarget(t *testing.T) {
```

Comment:
🥇

Reply:
Thanks for chacking this. Let's see what @ishangulhane or @TalAviel will say. I will soon start working on extending session store. Currently it stores all sessions in `sync.Map` in the adapter without any expiration (you can see it by searching for `w.tokenCache`). This needs to be extended in some plug-gable manner, through interface into more implementations. One of them could be Redis, the other one securecookie or something like that. This will have separate PR and/or issue.